spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/tpcds/datagen/DefaultParquetTransformer.java

```diff
@@ -24,13 +26,22 @@
 public final class DefaultParquetTransformer implements ParquetTransformer {
     @Override
     public void transform(
-            SparkSession sparkSession, StructType schema, String sourcePath, String destinationPath, String delimiter) {
-        Dataset<Row> tableDataset = sparkSession
-                .read()
-                .format("csv")
-                .option("delimiter", delimiter)
-                .schema(schema)
-                .load(sourcePath);
-        tableDataset.write().format("parquet").save(destinationPath);
+            SparkSession sparkSession,
+            StructType schema,
+            Set<String> sourcePaths,
+            String destinationPath,
+            String delimiter) {
+        Optional<Dataset<Row>> unioned = sourcePaths.stream()
+                .map(path -> sparkSession
+                        .read()
+                        .format("csv")
+                        .option("delimiter", delimiter)
+                        .schema(schema)
+                        .load(path))
```

Comment:
IIRC, doesn't `load` have a version that takes multiple files...?

Reply:
Yeah but we need to convert to a scala seq - not sure we want to add a scala dep for that?